examples/github_search/pubspec.yaml

```diff
@@ -41,9 +41,9 @@ dev_dependencies:
   build_runner: ^2.1.7
   # flutter_driver:
   #   sdk: flutter
+  flutter_lints: ^1.0.4
```

Comment:
I guess the version needs to be upgraded to 2.0.0 ?

Reply:
Dependencies updated!